verification/curator-service/api/src/model/source.ts

```diff
@@ -8,6 +8,24 @@ import { OriginDocument, originSchema } from './origin';
 import mongoose from 'mongoose';
 import { uploadSchema, UploadDocument } from './upload';
 
+const dateFilterSchema = new mongoose.Schema({
```

Comment:
Can you put this in a separate file, like the remainder of sub-docs?

Having all of them in here is a mess, and it's simpler to have all of 'em out than have some included here.

Can do a separate PR if you prefer!

Reply:
Sure, done.